app/models/site.rb

```diff
@@ -46,7 +46,7 @@ def assign_code
     self.code ||= pick_next_available_code(available_codes: provider&.unassigned_site_codes)
   end
 
-  def description
+  def to_s
     "#{location_name} (code: #{code})"
```

Comment:
Keep it the same as course?

Ie 

Main (-) 

Vs 

Main (code: -) 

?

Reply:
I think I started with that but then `Foo (code: 1)` looked really weird when the code is 0-9: `Foo (1)'